### DISCUSSION TITLE
Postgres SQL task: table name as a parameter causes SQL parsing error

### DISCUSSION THREAD

**Discussion by zhuyt1090** [apache/dolphinscheduler#11568] [Q&A]
![image](https://user-images.githubusercontent.com/108874436/185578863-ff41f963-e1e3-469a-94dc-176f75690f12.png)

![image](https://user-images.githubusercontent.com/108874436/185579074-3e5a9187-89e0-47a8-9f6e-45f9309c06d4.png)

The table name has single quotation marks, which is not supported by Postgres

**Comment by DarkAssassinator** [comment]
due to PreparedStatement. The sql task use preparedStatement to set these params, but preparedStatement do not support table name. A table name can't be used as a parameter, it must be hard coded.
But i think that dolphinscheduler should add a text plain param type. If user select TEXTPLAIN, the task just replace it.
